worlds/tits_the_3rd/__init__.py

```diff
@@ -80,5 +73,28 @@ def set_rules(self) -> None:
         self.multiworld.completion_condition[self.player] = lambda _: True
 
     def pre_fill(self):
-        item = self.create_item(ItemName.bennu_defeat)
-        self.multiworld.get_location(LocationName.chapter1_boss_defeated, self.player).place_locked_item(item)
+        # For now hard code beating Bennu as victory
+        victory_item = self.create_item(ItemName.bennu_defeat)
+        self.multiworld.get_location(LocationName.chapter1_boss_defeated, self.player).place_locked_item(victory_item)
+
+        # Randomize starting characters here
+        # TODO: Maybe add some options regarding this
```

Comment:
We can probably add this to this PR. should be as simple as `if self.options.<option_name>.value` after adding the option.